build.sbt

```diff
@@ -726,6 +726,12 @@ lazy val blazeServer = libraryProject("blaze-server")
         .exclude[DirectMissingMethodProblem]("org.http4s.blaze.server.BlazeServerBuilder.this"),
       ProblemFilters
         .exclude[DirectMissingMethodProblem]("org.http4s.blaze.server.WebSocketDecoder.this"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.http4s.blazecore.util.BodylessWriter.this"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.http4s.blazecore.util.BodylessWriter.ec"),
+      ProblemFilters
+        .exclude[DirectMissingMethodProblem]("org.http4s.blazecore.util.EntityBodyWriter.ec"),
```

Comment:
I think these are Scala 3 only?

Reply:
They definitely got changed on every version, but they're private.  Do I have it backward?  Is it Scala 3 that enforces privates and Scala 2 lets them through?  Maybe that's why I didn't see this locally.

Reply:
Mima doesn't know how to recognize package-privates on Scala 3 yet*, that's why we never see these locally.

* Actually It's fixed, but not released 😕 
- https://github.com/lightbend/mima/issues/682